### DISCUSSION TITLE
How to trigger Homekit Automation when Doorbell button is pressed?

### DISCUSSION THREAD

**Discussion by DavTechFr** [samemory/homebridge-eufy-security-control#299] [Q&A] [answered]
Hi All,

I’ve integrated an Eufy Security Doorbell 2K in Homekit thanks to this great plugin.

I’ve linked doorbell Event with HomePods, this works Fine, and I also receives notifications.

I also want to trigger HomeKit automation, however **I cannot found an accessory linked to this « Doorbell pressed » event.**
Available Doorbell accessories are:
- Motion « Doorbell »: I guess It activates when motion is detected 
- Switch « Doorbell Enabled » that is always On (seems useful to disable doorbell - « do not disturb »)
- Switch « Doorbell Motion » that is Off, I guess it turns On when motion is detected; I’ve deactivated this fonction on Eufy device.

Any Help?

Thanks
David

**Comment by DavTechFr** [comment]
Any help, PLEASE 

@schliemann @lenoxys

**Reply by ivangotti** [reply]
I am also struggling with that. I see no doorbell button in homekit.

**Reply by ivangotti** [reply]
@DavTechFr Could you find something? I don't have chime so my hope was to have the button press to trigger a sound in my speakers.

**Reply by schliemann** [reply]
Are you using Homepods? The possibilities of triggering sounds were previously relatively limited. But the automation itself can be achieved through eve or home+ 5.

**Comment by ivangotti** [comment]
I have a sonos but I can’t find a way to detect that the button was
pressed. How are you doing that?

Regards,
Ivan

On Mon, 27 Dec 2021 at 19:48, schliemann ***@***.***> wrote:

> Are you using Homepods? The possibilities of triggering sounds were
> previously relatively limited. But the automation itself can be achieved
> through eve or home+ 5.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/samemory/homebridge-eufy-security/discussions/299#discussioncomment-1876113>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALRAIDHKDS777PK2YQ7VA53UTCYIRANCNFSM5J4NPX2Q>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you commented.Message ID:
> <samemory/homebridge-eufy-security/repo-discussions/299/comments/1876113@
> github.com>
>
-- 
*Ivan Gotti*
www.linkedin.com/in/ivangotti

**Reply by schliemann** [reply]
You need a 3rd party app like eve or home+ 5.

**Comment by ivangotti** [comment, marked as answer]
Awesome! I can see the button now! Thanks you! 😄👍

On Mon, 27 Dec 2021 at 20:43, schliemann ***@***.***> wrote:

> You need a 3rd party app like eve or home+ 5.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/samemory/homebridge-eufy-security/discussions/299#discussioncomment-1876349>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALRAIDHM5PKZINDEOR6ZMG3UTC6XFANCNFSM5J4NPX2Q>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you commented.Message ID:
> <samemory/homebridge-eufy-security/repo-discussions/299/comments/1876349@
> github.com>
>
-- 
*Ivan Gotti*
www.linkedin.com/in/ivangotti
